### PR TITLE
Make `poetry run lnbits` use `.env` variables and fix uvicorn logs

### PR DESCRIPTION
**Pull request by dni**
this pr fixes this issue: https://github.com/lnbits/lnbits-legend/issues/785

* removes click and uses env variables instead

if i am not mistaken, click in this context with `poetry run lnbits` would always use it's default values (127.0.0.1:5000), which for me is unexpected and inconsistent with our specified ENV variables.

to make things worse the server.py would start with the click default values, but our logger would log our ENV. so it basically showed the wrong PORT and HOST.

### CODE REVIEW

**Comment by callebtc** [comment]
This PR removes `click`. Why? You did not update the poetry.lock and pyproject.toml, why?

**Comment by dni** [comment]
click in this context with `poetry run lnbits` would always use it's default values (127.0.0.1:5000), which for me is unexpected and inconsistent with our specified ENV variables. 

to make things worse the server.py would start with the click default values, but our logger would log our ENV. so it basically showed the wrong PORT and HOST.

File: lnbits/server.py [hunk 1]
```diff
@@ -1,18 +1,14 @@
...
 
-@click.command()
-@click.option("--port", default="5000", help="Port to run LNBits on")
-@click.option("--host", default="127.0.0.1", help="Host to run LNBits on")
-def main(port, host):
+
+def main():
     """Launched with `poetry run lnbits` at root level"""
-    uvicorn.run("lnbits.__main__:app", port=port, host=host)
+    config = uvicorn.Config("lnbits.__main__:app", port=PORT, host=HOST)
```
Comment (callebtc) [inline lnbits/server.py:8]: Can you extend this to pass all command line arguments to uvicorn?

**Review by callebtc — CHANGES_REQUESTED** [review]

File: lnbits/server.py (hunk 1 above)
Comment (callebtc) [reply lnbits/server.py:8]: This would also fix Issue #834.

**Comment by callebtc** [comment]
Fixed in #836.

**Pull request closed without merging**